Regression/Checksum/benchmarks_json/multi_J_rz_psatd.json

```diff
@@ -1,49 +1,45 @@
 {
   "driver": {
     "particle_cpu": 0.0,
-    "particle_id": 500000500000.0,
-    "particle_momentum_x": 8.710799591919723e-16,
-    "particle_momentum_y": 8.719557136998762e-16,
-    "particle_momentum_z": 5.461841477490683e-13,
-    "particle_position_x": 6.278329546764386,
-    "particle_position_y": 17.94664786026417,
-    "particle_theta": 1570796.3267948946,
-    "particle_weight": 6241509074.460753
+    "particle_id": 500000428326.0,
+    "particle_momentum_x": 8.723405122353729e-16,
+    "particle_momentum_y": 8.719285567351437e-16,
+    "particle_momentum_z": 5.461771334692466e-13,
+    "particle_position_x": 6.269566322411488,
+    "particle_position_y": 17.93420080596407,
+    "particle_theta": 1570790.043609588,
+    "particle_weight": 6241484108.424456
   },
   "lev=0": {
-    "Bx": 8.539565077847229e-12,
-    "By": 24912.146189783885,
-    "Bz": 4.086417276545147e-12,
-    "Ex": 4666716648235.921,
-    "Ey": 0.002579893581587712,
-    "Ez": 4313874828857.103,
-    "jx": 363242056086167.9,
-    "jy": 0.058699981448411834,
-    "jz": 1930576792579724.0,
-    "rho": 5286010.8543447815,
-    "rho_driver": 6281777.267757398,
-    "rho_plasma_e": 49565978.70143692,
-    "rho_plasma_p": 50769177.377112545
+    "By": 24912.66213622398,
+    "Ex": 4667305981763.072,
+    "Ez": 4307437907855.343,
+    "jx": 362736023475894.4,
+    "jz": 1937267309900551.0,
```

Comment:
Thanks for looking again into this, @NeilZaim! I see that you removed the checksums for `Bx` and `Bz` which are indeed close to zero (order `1e-12`), but I see that you also removed the ones for `Ey` and `jy` which are larger instead. I understand that physical units play a role here, but can you confirm that indeed the results were unstable also for `Ey` and `jy` (in other words, confirm that these were not removed by mistake)?

Reply:
Yes I confirm that `Ey` and `jy` are also unstable.

The value of `Ey` is the same order of magnitude as that of `Bx`and `Bz` (multiplied by `c`) and the value of `jy` is approximately 1e-15 that of `jx` & `jz` so also pretty much zero to machine precision.